### PR TITLE
About Page #4uaeb3

### PR DESCRIPTION
**Pull request by mahamzayed**
# Description

The purpose of this PR is to change the design of the About page to reflect SPARC's [designs](https://app.abstract.com/projects/e709ca90-5ba9-11e9-9eaa-59df656d954a/branches/7ff6a970-9a3d-4f46-ae0a-d10925710fc1/collections/36a06d50-1809-4db8-9af8-98bd7de45053). Please note the following:

- Images are not being rendered on the page as per designs until SPARC includes the official images in contentful
- About copy text that is being pulled in is based on what is in contentful. The copy text in the designs are outdated and need to be updated.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Navigate to the about page. Everything should look and behave as expected, including tablet and mobile viewports.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by mahamzayed** [comment]
@cameronbaney I realize that I'm mixing between `rem` and `px` in the css. I tried to convert all the px for font-size, line-height etc to em but it was throwing things off so I kept it as is. Maybe it's the converter I'm using so if you have a better unit converter in mind please share!

**Comment by cameronbaney** [comment]
@mahamzayed I'm not sure what convertor you're using, but if you do (pixel size)/16 that will give you the `em` units. This may have been a situation between rem/em.

File: pages/about/index.vue [hunk 1]
```diff
@@ -2,20 +2,54 @@
...
+      <div class="subpage">
+        <div class="row">
+          <div class="col">
+            <!-- <img class="about-page-portal-img" src="https://via.placeholder.com/222" /> -->
+          </div>
+          <div class="subpage-col">
+            <p class="about-page-text" v-html="parseMarkdown(sparcPortal)" />
+            <!-- <img class="about-page-portal-img-mobile" src="https://via.placeholder.com/222" /> -->
+            <button class="about-teams-and-projects-button">
+              <nuxt-link
```
Comment (cameronbaney) [inline pages/about/index.vue]: This link and copy should come from contentful as well.

File: pages/about/index.vue [hunk 2]
```diff
@@ -2,20 +2,54 @@
...
     <div class="page-wrap container">
       <div class="subpage">
-        <el-row type="flex" justify="center">
-          <el-col :row="24">
-            <!-- eslint-disable vue/no-v-html -->
-            <!-- marked will sanitize the HTML injected -->
-            <div v-html="parseMarkdown(copy)" />
-          </el-col>
-        </el-row>
+        <div class="row">
```
Comment (cameronbaney) [inline pages/about/index.vue]: Nit: Why not use `el-row`?

Comment (mahamzayed) [reply pages/about/index.vue]: @cameronbaney I was using `el-row` and `el-col` but they were giving me trouble with spacing and alignment, especially when it came to mobile and tablet viewports. I just found it a lot easier to use the traditional css flex attributes.

**Comment by mahamzayed** [comment]
> Remove the hero image as it is not approved/purchased.

@cameronbaney I spoke with Qian about the image and she told me that I could retrieve it from the Wrike ticket in contentful, which is why I included it here.

File: pages/about/index.vue (hunk 1 above)
Comment (mahamzayed) [reply pages/about/index.vue]: I'll change the link, but the copy is coming from contentful. The About page has two fields, one called `overview` and the other `sparcPortal`

Comment (cameronbaney) [reply pages/about/index.vue]: I mean the actual label "View Teams & Projects" of thee button.

File: pages/about/index.vue (hunk 2 above)
Comment (cameronbaney) [reply pages/about/index.vue]: Sounds good, thanks for the info.

**Comment by mahamzayed** [comment]
@cameronbaney Addressed feedback

**Pull request merged**